index.js

```diff
@@ -33,4 +46,6 @@ function getRuleURI(ruleId) {
   };
 }
 
+getRuleURI.plugins = plugins;
```

Comment:
> I’ve exported it in case a module wants to use it to get the raw URLs.

It's not part of the current API, and I'm not sure what use that would be for a consumer?

> I could also extract the parsing logic into a separate file.

I don't really see a need for it, especially since that adds time to import the "logic file".